src/main/java/com/robinpowered/sdk/model/Invited.java

```diff
@@ -0,0 +1,7 @@
+package com.robinpowered.sdk.model;
+
+/**
+ * Interface for models that are invited to an {@link Event}.
+ */
+public interface Invited {
```

Comment:
Prefer something like `Invitable` here. The `-able` suffix means "capable of being invited". `Invited` implies they are already invited and it more of a synonym for `Invitee`.